src/test/sighash_tests.cpp

```diff
@@ -254,15 +253,19 @@ BOOST_AUTO_TEST_CASE(sighash_from_data)
 
           CValidationState state;
           if (tx.fOverwintered) {
-              if (tx.nVersion == 3 && tx.nExpiryHeight > TX_EXPIRY_HEIGHT_THRESHOLD) {
+              // Note that OVERWINTER_MIN_CURRENT_VERSION and OVERWINTER_MAX_CURRENT_VERSION
+              // are checked in IsStandardTx(), not in CheckTransactionWithoutProofVerification()
```

Comment:
Wait, I thought that the ``OVERWINTER_{MIN,MAX}_CURRENT_VERSION`` check was supposed to be a consensus rule, not a standard rule?

Reply:
Was confused with ``OVERWINTER_{MIN,MAX}_TX_VERSION``. This is correct.